src/katgpucbf/xbgpu/engine.py

```diff
@@ -326,6 +493,13 @@ def __init__(
 
         self._populate_sensors()
 
+    @property
+    def output(self) -> XOutput:
+        """Alias the parent-level :class:`Output` list."""
```

Comment:
I think I'd find that docstring confusing if I didn't already know what was going on. How about
```suggestion
        """The single :class:`Output` produced by this pipeline."""
```
(it might need a noqa because it's not in imperative voice, which I think is fine for property docstrings since they're pretending to be data and not code).